### PR TITLE
Fix broken alpha kops test

### PR DESCRIPTION
**Pull request by upodroid**
This change should fix https://testgrid.k8s.io/sig-cluster-lifecycle-kubeup-to-kops#ci-kubernetes-e2e-cos-gce-alpha-features

### CODE REVIEW

**Review by hakman — APPROVED** [review]

File: config/jobs/kubernetes/kops/build_jobs.py [hunk 1]
```diff
@@ -880,6 +880,7 @@ def generate_misc():
                    build_cluster="k8s-infra-prow-build",
                    extra_flags=[
                        "--image=cos-cloud/cos-105-17412-156-49",
+                       "--set=spec.networking.networkID=default",
```
Comment (hakman) [inline config/jobs/kubernetes/kops/build_jobs.py:883]: Just curious, why is this needed?
```suggestion
                       "--set=spec.networking.networkID=default",
```

Comment (upodroid) [reply config/jobs/kubernetes/kops/build_jobs.py:883]: It hides this bug

https://github.com/kubernetes/cloud-provider-gcp/issues/612
